### PR TITLE
Add JS classes compat data

### PR DESCRIPTION
**Pull request by Elchi3**
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes and sub pages.

- extends: I didn't include "array subclassing", I think it doesn't belong there (we are also not mentioning subclassing of other objects).
- constructor: I didn't include "default constructor", seems like it is the same version in Firefox and otherwise unknown. Should be included in basic support.

### CODE REVIEW

File: javascript/classes.json [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+            {
+              "version_added": "49"
+            },
+            {
+              "version_added": "42",
+              "notes": "Requires strict mode. Non-strict mode support is behind the flag \"Enable Experimental JavaScript\", disabled by default."
+            }
+          ],
+          "chrome_android": {
+            "version_added": "56"
```
Comment (teoli2003) [inline javascript/classes.json]: Where does this 56 comes from?

File: javascript/classes.json [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": [
+              {
+                "version_added": "49"
+              },
+              {
+                "version_added": "42",
+                "notes": "Requires strict mode. Non-strict mode support is behind the flag \"Enable Experimental JavaScript\", disabled by default."
```
Comment (teoli2003) [inline javascript/classes.json]: I think we should either, merge both entries:

"chrome": { 
  "version_added": "49",
  "notes": "From Chrome 42 to 48 was only working in strict mode"
}

or to make a subfeature:
"javascript"."classes"  (with "42"/no comment)
"javascript"."classes"."non_strict" (with 49, and the pref entry with 42)

(and same values of Fx/IE/Edge/Safari)

File: javascript/classes.json [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+              }
+            ],
+            "chrome_android": {
+              "version_added": "56"
```
Comment (teoli2003) [inline javascript/classes.json]: 56?

File: javascript/classes.json [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+        }
+      },
+      "extends": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Classes/extends",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": [
```
Comment (teoli2003) [inline javascript/classes.json]: Same comment as before

File: javascript/classes.json [hunk 5]
```diff
@@ -0,0 +1,244 @@
+{
+  "javascript": {
+    "classes": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Classes",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": [
```
Comment (teoli2003) [inline javascript/classes.json]: See comment below about strict mode.

File: javascript/classes.json [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+              {
+                "version_added": "49"
+              },
+              {
+                "version_added": "42",
+                "notes": "Requires strict mode. Non-strict mode support is behind the flag \"Enable Experimental JavaScript\", disabled by default."
+              }
+            ],
+            "chrome_android": {
+              "version_added": "56"
```
Comment (teoli2003) [inline javascript/classes.json]: 56?

File: javascript/classes.json [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
```
Comment (teoli2003) [inline javascript/classes.json]: 56?

File: javascript/classes.json (hunk 1 above)
Comment (Elchi3) [reply javascript/classes.json]: No idea, I'm putting `true` here and below.

File: javascript/classes.json (hunk 2 above)
Comment (Elchi3) [reply javascript/classes.json]: I'm merging the two entries.

**Comment by Elchi3** [comment]
Updated!

**Pull request merged**